### PR TITLE
Release MongoDB Kubernetes Operator v0.11.0

### PR DESCRIPTION
**Pull request by mateigrigore**
### Summary:

helm changes: [mongodb/helm-charts#340](https://github.com/mongodb/helm-charts/pull/340)

### All Submissions:

* [x] Have you opened an Issue before filing this PR?
* [x] Have you signed our [CLA](https://www.mongodb.com/legal/contributor-agreement)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
* [x] Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).

### CODE REVIEW

File: docs/RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,19 +1,18 @@
-# MongoDB Kubernetes Operator 0.10.0
+# MongoDB Kubernetes Operator 0.11.0
 
-## Released images signed
+## Migrating agent images to ubi
+All agent images were updated to use the ubi repo
 
-All container images published for the community operator are signed with our private key. This is visible on our Quay registry. Signature can be verified using our public key, which is available at [this address](https://cosign.mongodb.com/mongodb-enterprise-kubernetes-operator.pem).
+## Documentation improvements
+Improvements were made to the documentatio of using the community operator as well as the one for local development.
```
Comment (Julien-Ben) [inline docs/RELEASE_NOTES.md:7]: Small typo here

File: docs/RELEASE_NOTES.md [hunk 2]
```diff
@@ -1,19 +1,18 @@
...
+- Added `AuditLogRotate` field to `AgentConfiguration`
+- Fixed JSON key to be lower case: `logRotate` 
 
-## Overriding Mongod settings via the CRD 
-- Example can be found [here](../config/samples/mongodb.com_v1_mongodbcommunity_override_ac_setting.yaml).
-
-## ReadinessProbe error logging
-- fixed a red herring which caused the probe to panic when the health status is not available. Instead it will just log the error
+## Bug Fixes
+- Users removed from the resource are now also deleted from the database and their connection string secretes are cleaned up
```
Comment (Julien-Ben) [inline docs/RELEASE_NOTES.md:14]: Typo here as well (secretes)

**Review by mircea-cosbuc — APPROVED** [review]

File: docs/RELEASE_NOTES.md [hunk 3]
```diff
@@ -1,19 +1,18 @@
...
+- Colisions of the scram secret name will now be spotted by spec validation
```
Comment (Julien-Ben) [inline docs/RELEASE_NOTES.md:15]: Typo as well here (spelling)
I would rephrase it with something like "Scram secret name collisions will now be caught when validating specs."

**Review by Julien-Ben — APPROVED** [review]
Congratulations for your first release ! 
Left a few comments regarding spelling and syntax on the release notes

Otherwise LGTM, well done

File: release.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "golang-builder-image": "golang:1.22",
-  "operator": "0.10.0",
+  "operator": "0.11.0",
   "version-upgrade-hook": "1.0.9",
   "readiness-probe": "1.0.20",
   "agent": "107.0.1.8507-1",
```
Comment (mircea-cosbuc) [inline release.json:6]: The agent should be bumped to `107.0.7.8596-1` as you've already bumped it in the helm chart https://github.com/mongodb/helm-charts/pull/332/files

**Review by mircea-cosbuc — APPROVED** [review]
LGTM!
